pkg/ccl/backupccl/testdata/backup-restore/in-progress-import-missing-data

```diff
@@ -0,0 +1,237 @@
+exec-sql
+RESTORE DATABASE d FROM LATEST IN 'nodelocal://0/database/';
+----
+pq: table "foo" cannot be safely restored from this backup. This backup is affected by issue #88042, which produced incorrect backups after an IMPORT. To continue the restore, you can either: 1) restore to a system time before the import completed, 2022-10-03 20:03:58.623676398 +0000 UTC; 2) restore with a newer backup chain (a full backup [+ incrementals]) taken after the current backup target; 3) or remove table foo from the restore targets.
+HINT: See: https://www.cockroachlabs.com/docs/advisories/a88042
+
+
+exec-sql
+RESTORE TABLE d.goodfoo FROM LATEST IN 'nodelocal://0/database/' with into_db = defaultdb;
+----
+pq: table "goodfoo" cannot be safely restored from this backup. This backup is affected by issue #88042, which produced incorrect backups after an IMPORT. To continue the restore, you can either: 1) restore to a system time before the import completed, 2022-10-03 20:03:58.623676398 +0000 UTC; 2) restore with a newer backup chain (a full backup [+ incrementals]) taken after the current backup target; 3) or remove table goodfoo from the restore targets.
+HINT: See: https://www.cockroachlabs.com/docs/advisories/a88042
+
+exec-sql
+RESTORE TABLE d.foofoo FROM LATEST IN 'nodelocal://0/database/' with into_db = defaultdb;
+----
+pq: table "foofoo" cannot be safely restored from this backup. This backup is affected by issue #88042, which produced incorrect backups after an IMPORT. To continue the restore, you can either: 1) restore to a system time before the import completed, 2022-10-03 20:03:58.623676398 +0000 UTC; 2) restore with a newer backup chain (a full backup [+ incrementals]) taken after the current backup target; 3) or remove table foofoo from the restore targets.
+HINT: See: https://www.cockroachlabs.com/docs/advisories/a88042
+
+
+# Ensure the restore succeeds if there are no corrupt tables in the target
+exec-sql
+RESTORE TABLE d.baz FROM LATEST IN 'nodelocal://0/database/' with into_db = defaultdb;
+----
+
+
+# Check that the wildcard restore encounters the same errors
+
+######################
+# Check that you can restore AOST while the tables are offline
+exec-sql
+RESTORE DATABASE d FROM LATEST IN 'nodelocal://0/database/' AS OF SYSTEM TIME '2022-10-03 20:03:58.623676398';
+----
+
+
+query-sql
+SELECT table_name FROM [SHOW TABLES FROM d];
+----
+baz
+
+
+###########################
+# Check the alternative scenarios
+###########################
+
+exec-sql
+DROP DATABASE d CASCADE;
+----
+
+# Check that a backup chain with the intermediate incremental backup restores properly
+exec-sql
+RESTORE DATABASE d FROM LATEST IN 'nodelocal://0/database_double_inc/';
+----
+
+query-sql
+SELECT count(*) FROM d.foo;
+----
+0
+
+
+query-sql
+SELECT count(*) FROM d.foofoo;
+----
+1
+
+query-sql
+SELECT count(*) FROM d.goodfoo;
+----
+3
+
+
+# Check that that backup chain with the intermediate incremental backup restores properly
```

Comment:
nit: I think we need to say backup without revision history here?

Reply:
ah. good catch!